cfgov/templates/wagtailadmin/account/password_reset/confirm.html

```diff
@@ -1,19 +1,19 @@
 {% extends "wagtailadmin/admin_base.html" %}
-{% load compress static i18n %}
+{% load static i18n %}
```

Comment:
Why is this `static` here but [`staticfiles` in Wagtail](https://github.com/wagtail/wagtail/blob/v1.13.1/wagtail/wagtailadmin/templates/wagtailadmin/account/password_reset/confirm.html#L2)? Looks like that was changed in Wagtail [here](https://github.com/wagtail/wagtail/commit/2444ee6e8d8ad2c3729e74cd73c93a6146261d1e#diff-c9bbbc9e5782ca30e7bcc08627d7629d).